aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -156,16 +156,12 @@ public void testS3FileIOWithS3FileIOAwsClientFactoryImpl() throws Exception {
   }
 
   @Test
-  public void testS3FileIOWithDefaultAwsClientFactoryImpl() throws Exception {
+  public void testS3FileIOWithAwsClientFactoryImpl() throws Exception {
```

Comment:
I think the naming was correct. The default factory that eventually will be used is `DefaultAwsClientFactory` if no other has been specified

Reply:
@nastra reverted in https://github.com/apache/iceberg/pull/9541/commits/bf71284ca8aa08d7ef066837619de16c9c8b683c